tests/test_event_loop_cleanup.py

```diff
@@ -0,0 +1,38 @@
+from textwrap import dedent
+
+
+def test_task_canceled_on_test_end(testdir):
```

Comment:
Great job!

At first the test didn't seem to fail when I removed the new cleanup code. Then I noticed that we're testing with pytest-6.2.5. When I bumped the pytest version the test failed correctly.

Could you replace `pytest == 6.2.5` with `pytest >= 6.2.5` in the `[testenv]` of `tox.ini` while you're at it?

Reply:
Hmm, that is strange. I'm running 7.1.1 locally.

Do I need to bump it in `[testenv:pytest-min]` too?

Reply:
The `pytest-min` environment was added to test pytest-asyncio against the lowest supported version of pytest. We shouldn't bump it without a proper deprecation warning well ahead of time.